src/metadata/persistence/comments.clj

```diff
@@ -122,9 +134,11 @@
    Parameters:
      comment-id - The UUID of the comment being readmitted."
   [comment-id]
-  (sql/update :comments
-    (set-fields {:retracted false :retracted_by nil})
-    (where {:id comment-id}))
+  (jsql/update! ds
+                (t "comments")
+                {:retracted false
+                 :retracted_by nil}
+                {:id comment-id})
```

Comment:
Hopefully fairly self-explanatory how the update works. After DB and table is updates, then conditions.